.eslintrc.js

```diff
@@ -2,7 +2,11 @@ module.exports = {
   env: {
     node: true
   },
-  extends: ['digitalbazaar'],
+  extends: [
+    'digitalbazaar',
+    'eslint-config-digitalbazaar',
```

Comment:
I believe you can drop `eslint-config-digitalbazaar` and just use the shortname `digitalbazaar` here.